src/atoms/TextInput/TextInput.native.js

```diff
@@ -182,8 +199,13 @@ const TextInput = ({
   return (
     <Flex justifyContent="flex-end">
       <View>
+        {!hasAnimatedLabel && labelPosition === 'top' ? (
```

Comment:
what if we just write `labelPosition==='top'`. Any reason you explicitly need to check `!hasAnimated`?

Reply:
label position can be 'top' and still have an animated label for outlined